test/e2e/framework/framework.go

```diff
@@ -260,6 +260,20 @@ func DetectGlobalnet() {
 	})
 }
 
+func FetchClusterNodes() error {
```

Comment:
The name is a bit confusing as you're actually initializing number of nodes per cluster, I'd change it to something clearer

Reply:
Its called as part of init method as you can see here - https://github.com/submariner-io/submariner/pull/1575 
where it's updating the cluster nodes. What name would you prefer?

Reply:
]```suggestion
func InitNumClusterNodes() error {
```

Reply:
Done